x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -163,6 +163,18 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
                     </Fragment>
                   </EuiFlexItem>
                 ) : null}
+                <EuiButtonEmpty
```

Comment:
The refresh button is slightly mis-aligned with the other buttons in the row
<img width="333" alt="Screen Shot 2021-01-07 at 10 59 37 AM" src="https://user-images.githubusercontent.com/13104637/103914183-71100c80-50d7-11eb-90af-c2c4beebc70c.png">

If you wrap it in an `EuiFlexItem` like the other buttons, that should fix this issue:
```
<EuiFlexItem grow={false}>
  <EuiButtonEmpty ...>
  </EuiButtonEmpty>
</EuiFlexItem>
```